src/functions.php

```diff
@@ -331,3 +331,15 @@ function json_encode($value, $options = 0, $depth = 512)
 
     return $json;
 }
+
+/**
+ * Wrapper for the hrtime() or microtime() functions
+ * (depending on the PHP version, one of the two is used)
+ *
+ * @return float|mixed UNIX timestamp
+ * @internal
+ */
+function current_time()
```

Comment:
One minor thing. All our other internal methods starts with an underscore. Please rename to `_current_time()`